eng/scripts/Language-Settings.ps1

```diff
@@ -90,3 +90,19 @@ function Get-AllPackageInfoFromRepo ([string] $ServiceDirectory) {
 
   return $allPackageProps
 }
+
+function Get-rust-EmitterName() {
+  return "@azure-tools/typespec-rust"
+}
+
+function Get-CrateVersion([string]$projectDirectory) {
+  return "0.1.0"
```

Comment:
This is stubbed out. Not sure where you want to pull this from, but it is required by typespec-rust.